conf/gconf/cluster.conf.sample

```diff
@@ -0,0 +1,257 @@
+# This is CORTX GConf (Config Store) which serves as input 
+# to all the CORTX Services
+# Location: /etc/cortx/cluster.conf 
+
+cortx:
+  external:
+    consul:
+      admin: admin
+      endpoints:
+      - http://consul-server.cortx-cluster.com:80
+      secret: MG12NU5BVjBVeWpBSWJuSV9uVjV5UzUxN3lTSVNkNUk1YkRzbjd2QkE9PQ==
+    kafka:
+      admin: admin
+      endpoints:
+      - kafka-server1.cortx-cluster.com
+      - kafka-server2.cortx-cluster.com
+      secret: TmRPNThMSm5GOVl5RHgwTVFfaTRJZ3k3YUUwMldmNkdxRE1KYVIxc0E9PQ==
+      
+  common:
+    product_release: LC              # TBD: To be removed
+    security:
+      device_certificate: /etc/cortx/solution/ssl/stx.pem
+      domain_certificate: /etc/cortx/solution/ssl/stx.pem
+      ssl_certificate: /opt/seagate/cortx/s3/install/haproxy/ssl/s3.seagate.com.pem
+    service:
+      admin: admin
+      secret: Z0FBQUFBQmhVY1dCdzJ6RkRXT3FSVjJXTWtUM0paZlB1LXFRRW0xNy1rRGhJLVpHUjcxMkVOeDZ6
+        MG9KWmpsM21yTmprZTJkYTBHUU1OWVl4dnZSaVRTeEs4UEdReHhkR1E9PQ==
+    setup_size: small                # TBD: To be removed and derived from limits
+    setup_type: K8                   # To be removed
+    storage:
+      config: /etc/cortx
+      local: /etc/cortx
+      log: /share/var/log/cortx/
+      shared: /share                 # To be removed
+      
+  csm:
+    agent:
+      endpoints:
+      - https://csm.seagate.com:8081
+    email_address: cortx@seagate.com # Optional 
+    mgmt_admin: cortxadmin
+    mgmt_secret: UlU1U0NrYklraS0ta3BOamN5X2J1VmUwRnJXTUFlb2JDdkItY1B3ZEE9PQ==
+      
+  s3:
+    auth_admin: admin
```

Comment:
add one key before "auth_admin", i.e.
```
s3:
  auth_server: auth_server                        # Auth User 
  auth_admin: admin
```

Reply:
@ujjwalpl  is this structure correct ?
s3 :
    auth_admin : admin
    auth_admin_access: access-key-plain-text
    auth_admin_secret:<binary of plain text secret key generated by kubernetes secret.yaml>